easybuild/easyconfigs/s/snpEff/snpEff-5.0-Java-13.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '5.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://pcingola.github.io/SnpEff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+ It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://snpeff.blob.core.windows.net/versions/']
+sources = [('%s_latest_core.zip' % (name))]
```

@verdurin better to include the version in the source filename, to avoid that newer versions overwrite older ones:
```suggestion
sources = ['%%(name)s_v%s_core.zip' % version.replace('.', '_')]
```

---

@smoors I agree, but the official link doesn't have a version in it.

---

@verdurin it's a problem for you the link I suggest is not officially advertised? I'd say let's use it as long as it works, because using a link that is overwritten with new versions is even more problematic imho.